src/main/java/codesquad/codestagram/article/controller/ArticleController.java

```diff
@@ -40,12 +64,99 @@ public String getArticleList(Model model){
     }
 
 
-    // 게시글 상세 보기 구현하기
-    @GetMapping("/articles/{id}")
-    public String getArticleById(@PathVariable("id") Long id, Model model) {
+    // 게시글 상세 보기 구현하기 -> 로그인한 사용자만 게시글의 세부 내용을 볼 수 있다
+    @GetMapping("/articles/{articleId}")
+    public String getArticleById(@PathVariable("articleId") Long id, Model model, HttpSession session) {
+        if(SessionUtil.isUserLoggedIn(session)){
```

Comment:
```suggestion
        if(!SessionUtil.isUserLoggedIn(session)){
```
이렇게 하면 불필요한 else를 제거할 수 있을 거 같아요. 
else가 나쁜건 아니지만 꼭 필요하지 않다면 쓰지 않는게 가독성 면에서 유리해 보입니다.